File: scratch/inspect_cabling_tables.py
import os
import requests

env_file = ".env"
supabase_url = ""
supabase_key = ""

with open(env_file, "r") as f:
    for line in f:
        if line.startswith("VITE_SUPABASE_URL="):
            supabase_url = line.strip().split("=")[1].strip("'\"")
        elif line.startswith("VITE_SUPABASE_ANON_KEY="):
            supabase_key = line.strip().split("=")[1].strip("'\"")

headers = {
    "apikey": supabase_key,
    "Authorization": f"Bearer {supabase_key}"
}

print("ac_cabling_specs:")
res = requests.get(f"{supabase_url}/rest/v1/ac_cabling_specs?select=*", headers=headers)
print(res.json())

print("dc_twin_cabling_specs:")
res = requests.get(f"{supabase_url}/rest/v1/dc_twin_cabling_specs?select=*", headers=headers)
print(res.json())

File: scratch/inspect_cabling_tables.ts
import { createClient } from '@supabase/supabase-js';
import * as dotenv from 'dotenv';

dotenv.config({ path: '.env.local' });
const supabaseUrl = process.env.VITE_SUPABASE_URL || '';
const supabaseKey = process.env.VITE_SUPABASE_ANON_KEY || '';
const supabase = createClient(supabaseUrl, supabaseKey);

async function check() {
  const { data: ac, error: e1 } = await supabase.from('ac_cabling_specs').select('*').limit(5);
  console.log('ac_cabling_specs:', ac, e1);
  const { data: dc, error: e2 } = await supabase.from('twin_dc_cabling_specs').select('*').limit(5);
  console.log('twin_dc_cabling_specs:', dc, e2);
}
check();
